src/PIL/QoiImagePlugin.py

```diff
@@ -0,0 +1,106 @@
+#
+# The Python Imaging Library.
+# $Id$
```

Comment:
We could skip this for new files:
```suggestion
```

Reply:
Do you have any ideas what the original meaning was? Is this some sort of placeholder?

Reply:
It's old VCS stuff, they'd replace that when checking out.

https://stackoverflow.com/q/2763070/724176

Reply:
Thanks, I've removed it.

Reply:
Is there any downside to removing it for all existing files as well? https://github.com/python-pillow/Pillow/pull/7009

Reply:
Just the downsides of a big chunk of code churn. Perhaps we remove it incrementally as we touch the files for other reasons?

Reply:
Ok, thanks. I've closed the PR.